core/src/main/kotlin/org/evomaster/core/problem/rest/oracle/RestSecurityOracle.kt

```diff
@@ -53,6 +53,63 @@ object RestSecurityOracle {
         }
     }
 
+    fun hasForgottenAuthentication(
+        path: RestPath,
+        individual: RestIndividual,
+        actionResults: List<ActionResult>
+    ): Boolean{
+
+        verifySampleType(individual)
+
+        val actions = individual.seeMainExecutableActions().filter {
+            it.path == path
```

Comment:
hmmm... actually this should be on the endpoint (verb+path), instead of just path. you might need auth for a POST to create data, but none for GET of public data